### ISSUE TITLE
Setup different settings files (development, production)

### ISSUE THREAD

**Issue by sascha11110** [Integreat/integreat-cms#74]
Use DJANGO_SETTINGS_MODULE as corresponding environment variable 

* remove EMAIL_BACKEND and setup a production ready mail server
* set DEBUG to false
* django won't handle static files anymore

**Comment by rasmus-storjohann-PG** [comment]
When we got started with Django, we used the [cookiecutter](https://cookiecutter-django.readthedocs.io/en/latest/) bootstrap tool. It has a lot of options but it is worth the time to get it right at the outset, because building all that tech ourselves would have been a lot of work: setting up testing tools, local/test/prod environments, docker, and much more. This is [the PR](https://github.com/pg-irc/pathways-backend/issues/113) for this work, see particularly [this commit](https://github.com/rasmus-storjohann-PG/pathways-backend/commit/3fbce288318864bb5cbadcb28130b32960531b43) but also the following ones to take the output from cookiecutter and making it the way we wanted it. We already had come code running at the time we did this, so it got more complicated to merge our existing code and the cookiecutter output, but it worked and still saved us a lot of time. Using it at the outset would have been a lot simpler.
